site2/docs/getting-started-standalone.md

```diff
@@ -4,25 +4,56 @@ title: Set up a standalone Pulsar locally
 sidebar_label: Run Pulsar locally
 ---
 
-For local development and testing, you can run Pulsar in standalone mode on your machine. The standalone mode includes a Pulsar broker, the necessary ZooKeeper and BookKeeper components running inside of a single Java Virtual Machine (JVM) process.
+## Preparation
 
-> #### Pulsar in production? 
-> If you're looking to run a full production Pulsar installation, see the [Deploying a Pulsar instance](deploy-bare-metal.md) guide.
+### Requirements
 
-## Install Pulsar standalone
+Currently, Pulsar is available for 64-bit **macOS**, **Linux**, and **Windows**. To use Pulsar, you need to install 64-bit JRE/JDK 8 or later versions.
 
-This tutorial guides you through every step of the installation process.
+> **Note**  
+> Broker is only supported on 64-bit JVM.
 
-### System requirements
+> **Tip**  
+> By default, Pulsar allocates 2G JVM heap memory to start. It can be changed in `conf/pulsar_env.sh` file under `PULSAR_MEM`. This is extra options passed into JVM. 
 
-Currently, Pulsar is available for 64-bit **macOS**, **Linux**, and **Windows**. To use Pulsar, you need to install 64-bit JRE/JDK 8 or later versions.
+For local development and testing, you can run Pulsar in standalone mode on your machine. The standalone mode includes a Pulsar broker, the necessary ZooKeeper and BookKeeper components running inside of a single Java Virtual Machine (JVM) process.
 
-> #### Tip
-> By default, Pulsar allocates 2G JVM heap memory to start. It can be changed in `conf/pulsar_env.sh` file under `PULSAR_MEM`. This is extra options passed into JVM. 
+> **Note**  
+> You can reuse existing Zookeeper clusters.
 
-> **Note**
-> 
-> Broker is only supported on 64-bit JVM.
+> **Pulsar in production?**  
+> If you're looking to run a full production Pulsar installation, see the [Deploying a Pulsar instance](deploy-bare-metal.md) guide.
+
+### Hardware considerations
```

Comment:
Why need to add hardware considerations (which are usually added to the `deployment` chapter ) to the `getting started` guide?